registry/client/repository.go

```diff
@@ -251,6 +251,11 @@ func (ms *manifests) Delete(dgst digest.Digest) error {
 	}
 }
 
+func (ms *manifests) Verify(ctx context.Context, manifest *manifest.SignedManifest) error {
+	// implemented and called in remote service
```

Comment:
We should still do some verification here, or this should be guaranteed by the manifest service.